components/layout/section/section.js

```diff
@@ -1,38 +1,32 @@
 import PropTypes from 'prop-types'
-import { Box, Grid, Typography } from '@mui/material'
+import { Box, Typography, Stack } from '@mui/material'
 
 export const Section = ({ title, children }) => {
   return (
-    <Grid
-      container
-      columns={ 8 }
-      columnSpacing={{ xs: 0, md: 2 }}
-      rowSpacing={{ xs: 2, md: 10 }}
+    <Stack
+      direction="row"
```

Comment:
let's build responsiveness into this component, and stack the title and body on smaller displays